src/images/icons/quiz_1.svg

```diff
@@ -0,0 +1,18 @@
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="512.000000pt" height="512.000000pt" viewBox="0 0 512.000000 512.000000" preserveAspectRatio="xMidYMid meet">
```

Comment:
Why do we have both png & svg?
Also the svg should not have any fill color & color should be done using css so that we can use 1 svg file to use it for different colors.

Reply:
@govind007?